codegen/util/packages.go

```diff
@@ -42,3 +44,9 @@ func GeneratedGoPackage(grp model.Group) string {
 
 	return s
 }
+
+// Generate a package alias, eg 'import alias github.com/my/package'
+// TODO:  Do something prettier if this works
```

Comment:
Is this TODO to be addressed in this PR? If not, maybe we should add an issue for this and reference it here.

Reply:
No, it's not.  Ultimately, it's just a style thing.  I think creating an issue makes sense, then we can decide if it's worth it for increased readability of the types.go file